src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -35,14 +35,18 @@ export class ApiInterceptorService implements HttpInterceptor {
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     const defaultVersion = '1.0';
+    let version = request.headers.get('version');
+    if (!version) {
+      version = defaultVersion;
+    }
```

Comment:
I have a suggestion above. Consider it as just a suggestion.

Reply:
The approach has been changed.